src/imitation/policies/exploration_wrapper.py

```diff
@@ -0,0 +1,64 @@
+"""Wrapper to turn a policy into a more exploratory version."""
+
+from typing import Optional
+
+import numpy as np
+from stable_baselines3.common import vec_env
+
+from imitation.data import rollout
+
+
+class ExplorationWrapper:
+    """Wraps a PolicyCallable to create a partially randomized version.
```

Comment:
I think we could make this a more general `MixtureWrapper` without much implementation complexity: take a sequence of `policies` and `probs` (defining a categorical probability distribution), and `stay_prob`.

`ExplorationWrapper` is then just special-case where `policies = [policy, self._random_policy]` and `probs = [random_prob, 1 - random_prob]`.

I don't have a concrete use case in mind for this, so think it's fine to leave it as-is, but feels like a feature that's nice to have around. E.g. what if we want a random policy with some different distribution over actions?

Reply:
I had the same thought but I think this should be easy to change later if we ever need it, so I'll stay with the simple  special case for now